python_modules/dagster/dagster/_core/pipes/utils.py

```diff
@@ -236,29 +242,15 @@ def no_messages_debug_text(self) -> str:
 WAIT_FOR_LOGS_TIMEOUT = 60
 
 
-class PipesBlobStoreMessageReader(PipesMessageReader):
-    """Message reader that reads a sequence of message chunks written by an external process into a
-    blob store such as S3, Azure blob storage, or GCS.
-
-    The reader maintains a counter, starting at 1, that is synchronized with a message writer in
-    some pipes process. The reader starts a thread that periodically attempts to read a chunk
-    indexed by the counter at some location expected to be written by the pipes process. The chunk
-    should be a file with each line corresponding to a JSON-encoded pipes message. When a chunk is
-    successfully read, the messages are processed and the counter is incremented. The
-    :py:class:`PipesBlobStoreMessageWriter` on the other end is expected to similarly increment a
-    counter (starting from 1) on successful write, keeping counters on the read and write end in
-    sync.
-
-    If `log_readers` is passed, the message reader will start the passed log readers when the
-    `opened` message is received from the external process.
+class PipesThreadedMessageReader(PipesMessageReader):
+    """A base class for MessageReaders which are supposed to read messages and logs in background threads.
```

Comment:
Since there is no actual class called "MessageReader", prefer:

"""A base class for message readers that read messages and logs in background threads."""